build-gluster-org/scripts/comment-on-open-pr.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+'''
+A small script to comment on opened PR for reviews on Gerrit
+'''
+import json
+import os
+import sys
+import requests
+
+r = requests.get('https://review.gluster.org/changes/'
+                     '?q=status:open+'
+                     'project:glusterfs')
+
+output = r.text
+    cleaned_output = '\n'.join(output.split('\n')[1:])
```

Comment:
I am a bit surprised, the indentation seems not correct.

Reply:
Ah, I thought I fixed it and already pushed with the changes!